tests/src/test/java/alluxio/rest/TestCase.java

```diff
@@ -13,12 +13,14 @@
 
 import alluxio.Constants;
 
+import com.amazonaws.util.IOUtils;
```

Comment:
this seems like an odd dependency to use here ... any chance the Apache Commons library provide equivalent functionality?